df_chat/drf/serializers.py

```diff
@@ -182,6 +202,10 @@ class RoomSerializer(CreatorMixin, serializers.ModelSerializer):
     )
     last_message = serializers.SerializerMethodField()
     is_muted = serializers.BooleanField(read_only=True)
+    category_rooms_set = CategorySerializer(
```

Comment:
This code will show all room categories for all Users. We need to include own categories only

Reply:
> This code will show all room categories for all Users. We need to include own categories only

Where should user be able to edit category for each room? At `api/v1/chat/categories/` only or at both, previous one plus `api/v1/chat/rooms/`? In first case, `RoomSerializer.category` should be changed to `SerializerMethodField` with 
```` 
def get_categories(self, obj):
    user = self.context['request'].user
    categories = obj.categories.filter(owner=user)
    serializer = CategorySerializer(categories, many=True)
    return serializer.data
````
Is it OK?